brownie/network/rpc.py

```diff
@@ -94,14 +96,22 @@ def launch(self, cmd: str, **kwargs: Dict) -> None:
             kwargs["evm_version"] = EVM_EQUIVALENTS[kwargs["evm_version"]]  # type: ignore
         kwargs = _validate_cmd_settings(kwargs)
         for key, value in [(k, v) for k, v in kwargs.items() if v]:
-            try:
-                cmd_list.extend([CLI_FLAGS[key], str(value)])
-            except KeyError:
-                warnings.warn(
-                    f"Ignoring invalid commandline setting for ganache-cli: "
-                    f'"{key}" with value "{value}".',
-                    InvalidArgumentWarning,
-                )
+            if key == "unlock":
+                if not isinstance(value, list):
+                    value: [value]  # type: ignore
+                for address in value:
+                    if isinstance(address, int):
+                        address = HexBytes(address.to_bytes(20, "big")).hex()
+                    cmd_list.extend([CLI_FLAGS[key], str(address)])
```

Comment:
is the conversion to `str` necessary?  I don't know if you can give any types in yaml other than `str` or `int` and you've handled integers already.